dashboards/bigquery/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+id: resource_list/bigquery_dataset
```

Comment:
Can you rename the directory from `bigquery` to `google-bigquery` to match the convention used by the other Google services dashboards?

Reply:
moved files to google-bigquery.